### PR TITLE
Bypass admin log for new users (i.e. only log when updating the permission)

### PR DESCRIPTION
**Pull request by aoby**

### CODE REVIEW

File: dashboard/app/models/user.rb [hunk 1]
```diff
@@ -248,6 +248,8 @@ def delete_course_as_facilitator(course)
   end
 
   def log_admin_save
+    return if new_record? && !admin?
```
Comment (ashercodeorg) [inline dashboard/app/models/user.rb]: Actually, should this belong in the `log_admin_save` method or in the `before_save` callback logic?

Comment (aoby) [reply dashboard/app/models/user.rb]: Good point. We should be consistent, rather than splitting up the logic. I'll move it to the callback

**Comment by ashercodeorg** [comment]
LGTM (e175cff).

**Comment by ashercodeorg** [comment]
Merging pre-Circle to unblock `test`.

**Pull request merged**